### PR TITLE
MOO-341: Add Register Patient to the Search widget

### PR DESCRIPTION
**Pull request by ssmusoke**
- Disable coreapps search widget and replacing it with an MSF one
- Rename the msfcore_auditlogs_app.json to msf_homepage_app.json to reflect the contents

https://jembiprojects.jira.com/projects/MOO/issues/MOO-341

### CODE REVIEW

File: omod/src/main/resources/apps/msfcore_auditlogs_app.json [hunk 1]
```diff
@@ -1,30 +0,0 @@
-[
-  {
```
Comment (kaweesi) [inline omod/src/main/resources/apps/msfcore_auditlogs_app.json]: i prefer we maintain this app separately, maybe just renamed to msfcore_admin_app.json but the register app should be a different one

**Comment by kaweesi** [comment]
thanks @ssmusoke, i don't see the new widget or config injection for add register patient

File: api/src/main/java/org/openmrs/module/msfcore/MSFCoreActivator.java [hunk 2]
```diff
@@ -61,6 +61,8 @@ public void started() {
     private void installMSFMeta() {
         log.info("Replacing default reference application registration app");
         Context.getService(AppFrameworkService.class).disableApp(MSFCoreConfig.REGISTRATION_APP_EXTENSION_ID);
+        // disable the default find patient app to provide one which allows searching for patients at the footer of the search for patients page
+        Context.getService(AppFrameworkService.class).disableApp(MSFCoreConfig.SEARCH_APP_EXTENSION_ID);
```
Comment (kaweesi) [inline api/src/main/java/org/openmrs/module/msfcore/MSFCoreActivator.java]: also enable this when this module is stopped

File: omod/src/main/resources/apps/msfcore_auditlogs_app.json (hunk 1 above)
Comment (ssmusoke) [reply omod/src/main/resources/apps/msfcore_auditlogs_app.json]: Okay - however I figured you will end up with a proliferation of xx_app.json files without knowing where they are pointing. 

Hence:
- homepage_app.json (home page links and widgets) 
- patientdashboard_app.json (dashboard widgets)

File: api/src/main/java/org/openmrs/module/msfcore/MSFCoreActivator.java (hunk 2 above)
Comment (ssmusoke) [reply api/src/main/java/org/openmrs/module/msfcore/MSFCoreActivator.java]: Will do

**Comment by tmvumbi2** [comment]
Thanks @ssmusoke, the new button positioning is looking good :+1: ,
Is it possible to remove the button copy below the table? see screen: 
image

**Comment by ssmusoke** [comment]
@tmvumbi2 The hiding of the section is dependent on this PR being merged https://github.com/openmrs/openmrs-module-coreapps/pull/159 which enables this line to work https://github.com/jembi/openmrs-module-msfcore/blob/1d1fc9e2fa1a31b35183f7fc02e4fe4338b32971/omod/src/main/webapp/resources/scripts/msfcore.js#L5

**Comment by ssmusoke** [comment]
@kaweesi I am not sure how the commit https://github.com/jembi/openmrs-module-msfcore/pull/12/commits/b16c62521d3d0592924f1182f6e12593f99aa467 came into my PR - I just pulled from upstream master 

I am going to have to delete and resubmit

**Comment by ssmusoke** [comment]
Closing to fix strange commits

**Pull request closed without merging**